pkg/ocp4/ocp4.go

```diff
@@ -36,78 +23,27 @@ const OCP4InstallMsg = `To install OCP4 run the installer as follow in order to
 
 const manifestPrefix = "100_CPMA-cluster-"
 
-// Translate translate OCP3 configs
-func (ocp4Master *Master) Translate(ocp3Master configv1.MasterConfig) {
-	if ocp3Master.OAuthConfig != nil {
-		logrus.Debugln("Translating oauth config")
-		oauth, secrets, err := oauth.Translate(ocp3Master.OAuthConfig)
-
-		if err != nil {
-			logrus.WithError(err).Fatalf("Unable to generate OAuth CRD from %+v", ocp3Master.OAuthConfig)
-		}
-		ocp4Master.OAuth = *oauth
-		ocp4Master.Secrets = secrets
-	}
-
-	if &ocp3Master.NetworkConfig != nil {
-		logrus.Debugln("Translating SDN config")
-		networkCR := sdn.Translate(ocp3Master.NetworkConfig)
-		ocp4Master.Network = *networkCR
-	}
-}
-
-func (ocp4Node *Node) Translate(ocp3Node configv1.NodeConfig) {
-}
-
-// GenYAML returns the list of translated CRDs
-func (ocp4Master *Master) GenYAML() []Manifest {
-	var manifests []Manifest
-
-	// Generate yaml for oauth config
-	crd := ocp4Master.OAuth.GenYAML()
-	manifests = oauthManifests(ocp4Master.OAuth.Kind, crd, manifests)
-
-	// Generate yaml for oauth secrets
-	manifests = oauthSecrets(ocp4Master.Secrets, manifests)
-
-	// Generate yaml for SDN config
-	networkCR := ocp4Master.Network.GenYAML()
-	manifests = sdnManifest(networkCR, manifests)
-
-	return manifests
-}
-
-func oauthManifests(oauthCRKind string, crd []byte, manifests []Manifest) []Manifest {
+func OAuthManifest(oauthCRKind string, crd []byte, manifests Manifests) Manifests {
 	if oauthCRKind != "" {
 		filename := manifestPrefix + "config-oauth.yaml"
-		manifest := Manifest{Name: filename, CRD: crd}
-		manifests = append(manifests, manifest)
+		m := Manifest{Name: filename, CRD: crd}
+		manifests = append(manifests, m)
 	} else {
 		logrus.Debugln("Skipping oauth, no manifests found")
 	}
-
 	return manifests
 }
 
-func oauthSecrets(ocp4MasterSecrets []secrets.Secret, manifests []Manifest) []Manifest {
-	for _, secret := range ocp4MasterSecrets {
-		filename := manifestPrefix + "config-secret-" + secret.Metadata.Name + ".yaml"
-		m := Manifest{Name: filename, CRD: secret.GenYAML()}
-		manifests = append(manifests, m)
-	}
-
+func SecretsManifest(secret secrets.Secret, crd []byte, manifests Manifests) Manifests {
+	filename := manifestPrefix + "config-secret-" + secret.Metadata.Name + ".yaml"
```

Comment:
This is a pedantic comment, but `fmt.Sprintf` is a better alternative to adding strings to concatenate them.

Reply:
Although not my style too. ;)